### PR TITLE
Adding support for es3

### PR DESCRIPTION
**Pull request by ldayananda**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -60,7 +60,8 @@ module.exports = function(grunt) {
     },
     uglify: {
       options: {
-        banner: '<%= banner %>'
+        banner: '<%= banner %>',
+        screwIE8: false
```
Comment (gkatsev) [inline Gruntfile.js]: I think this has to be under mangle, unless grunt-uglify has that as a top level option.

Comment (incompl) [inline Gruntfile.js]: It's top level in the grunt-uglify docs

**Comment by incompl** [comment]
LGTM, approved for deploy to production after testing.

**Comment by bcvio** [comment]
LGTM

**Comment by rbalakrishna93** [comment]
QA LGTM

**Pull request merged**